renderer/models/dashboard/INodeInitializer.ts

```diff
@@ -2,7 +2,7 @@ import { FormType } from './FormType';
 
 export interface INodeInitializerForm {
   workspaceDir: string[];
-  formControls: FormControls;
+  formControls: FormControls | any;
```

Comment:
Be careful with any. Doing `something | any` is pointless because **any** is more general than **something**
So `(something | any) <-- are the same --> any`

Reply:
Previously I did not add it. While building you showed some issues while building the application. And also I have given a pull request for the build fix. But you did not even merge these changes. 

Let me know are you facing any build issues with the file javainitializer, nodeinitializer ?

![image](https://user-images.githubusercontent.com/29056094/91688888-aaac9600-eb80-11ea-82e3-89c43bc662e0.png)